crates/router/src/services.rs

```diff
@@ -23,21 +24,21 @@ use crate::{
 
 #[async_trait::async_trait]
 pub trait PubSubInterface {
-    async fn subscribe(&self, channel: &str) -> errors::CustomResult<(), redis_errors::RedisError>;
+    async fn subscribe(&self, channel: &str) -> CustomResult<(), redis_errors::RedisError>;
```

Comment:
Could you please revert this change and qualify `CustomResult` with `errors::CustomResult` instead (here and elsewhere in this file)? Adds a bit of unnecessary clutter to the changes in the PR.